### PR TITLE
samples: nrf_desktop: Enable USB support on PCA10056

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-450

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebug.conf [hunk 1]
```diff
@@ -53,6 +55,21 @@ CONFIG_PWM_NRFX=y
...
+CONFIG_USB_DEVICE_PRODUCT="Mouse nRF52 Desktop"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x52DE
+CONFIG_USB_DEVICE_HID=y
+CONFIG_USB_DEVICE_LOG_LEVEL_OFF=y
+CONFIG_USB_DRIVER_LOG_LEVEL_OFF=y
+CONFIG_USB_HID_POLL_INTERVAL_MS=1
+CONFIG_USB_HID_BOOT_PROTOCOL=y
+CONFIG_USB_HID_PROTOCOL_CODE=2
+
```
Comment (MarekPieta) [inline samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebug.conf]: 2 empty lines ?

File: samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebugWithShell.conf [hunk 2]
```diff
@@ -57,6 +59,21 @@ CONFIG_PWM_NRFX=y
...
+CONFIG_USB_DEVICE_PRODUCT="Mouse nRF52 Desktop"
+CONFIG_USB_DEVICE_VID=0x1915
+CONFIG_USB_DEVICE_PID=0x52DE
+CONFIG_USB_DEVICE_HID=y
+CONFIG_USB_DEVICE_LOG_LEVEL_OFF=y
+CONFIG_USB_DRIVER_LOG_LEVEL_OFF=y
+CONFIG_USB_HID_POLL_INTERVAL_MS=1
+CONFIG_USB_HID_BOOT_PROTOCOL=y
+CONFIG_USB_HID_PROTOCOL_CODE=2
+
```
Comment (MarekPieta) [inline samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebugWithShell.conf]: 2 empty lines ?

File: samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebug.conf (hunk 1 above)
Comment (pdunaj) [reply samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebug.conf]: Fixed

File: samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebugWithShell.conf (hunk 2 above)
Comment (pdunaj) [reply samples/nrf_desktop/configuration/nrf52840_pca10056_ZDebugWithShell.conf]: Fixed

**Comment by pdunaj** [comment]
Hi @pizi-nordic , could you merge?

**Pull request merged**